build/all/pre-bundle.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+set -e
+
+##
+# pre-download equinix metal provider, since it only works directly from the registry with terraform 0.13.0+
```

Comment:
Should we consider upgrading tf to `v0.13` instead for simplicity sake?
Now, that we have separate docker images for the different providers, we can choose the tf version per provider even.

Reply:
That certainly would make life much easier. I wasn't sure if it was one-per-provider in the official distros or not, and whether or not this needed to be 0.12.x. I chatted with @rfranzke via Slack, and I had understood (easily my error) that switching to 0.13.x is non-trivial. If so, this looked like a simple bridge.

Switching to 0.13 means that:

* change the TF version (simple)
* the hcl for each terraform-bundle is slightly different (the format has changed a bit)
* the tf code in each extension has to be compatible - but this shouldn't be an issue, since this is 0.12->0.13, which is backwards-compatible, not 0.11->0.12, which can be a headache

I don't think I can do it _just_ for Packet, because there is a `build/all/` repo which packages them all, so I assume they all need to be aligned.

I am happy to go whichever way is easier.

Reply:
> the tf code in each extension has to be compatible

I don't know, how much effort it would be to adapt all the provider extensions' `.tf` stuff, don't have the overview.

> I don't think I can do it just for Packet, because there is a build/all/ repo which packages them all, so I assume they all need to be aligned.

Additional option: you could also leave the `all` variant like it is (or even remove the `packet`) provider, as I assume you will only be using the `equinixmetal` variant in the EM extension, and only bump the `TF_VERSION` there.
If that's fine for you, it will probably be the easiest option.

The `all` variant is anyways kind of orphaned, I would say. It's not deprecated yet, but we might just consider marking it as such now.

Reply:
If I only have to do the packet one, sure, I can take care of that. I will update this PR with a second commit. If it looks good, comment, and I will squash them into one. Leaving two commit lets me roll back if the second variant doesn't work.

Reply:
Actually, no need. It worked like a charm. Pushing now.